### DISCUSSION TITLE
Disabling the save button after click?

### DISCUSSION THREAD

**Discussion by mlauchli** [piccolo-orm/piccolo_admin#395] [Q&A] [answered]
I have a few hooks configured that fire when a record is updated. These submit an API request to a third party, and obviously the response back is not instantaneous. Is there a way to disable the save button upon click to ensure no repeat saves are made?

**Comment by dantownsend** [comment]
It's a good point. We'll have to make a code change to the front end to disable the save button until a response gets back.

**Reply by dantownsend** [reply]
I've opened a PR to disable the save button until the network call completes https://github.com/piccolo-orm/piccolo_admin/pull/397

Will release it once it has been tested.

**Reply by dantownsend** [reply, marked as answer]
In `piccolo_admin==1.5.0` this should be fixed.
